symplyphysics/definitions/thermodynamic_compressibility.py

```diff
@@ -29,14 +29,14 @@
 :symbols:`thermodynamic_compressibility` of the gas.
 """
 
-volume = clone_as_function(symbols.volume, display_symbol="V(p)")
+pressure = symbols.pressure
 """
-:symbols:`volume` of the gas as a function of pressure and other parameters.
+:symbols:`pressure` in the gas.
 """
 
-pressure = symbols.pressure
+volume = clone_as_function(symbols.volume, [pressure])
 """
-:symbols:`pressure` in the gas.
+:symbols:`volume` of the gas as a function of pressure and other parameters.
```

Comment:
You can artificially add other parameters and let autogenerate the result.

Reply:
Can you elaborate?

Reply:
Smth like `parameters = SymbolNew("p", any_dimension)`